internal/attachment/repository/mysql/mysql_attachment.go

```diff
@@ -0,0 +1,42 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/meroedu/meroedu/internal/domain"
+	"github.com/meroedu/meroedu/pkg/log"
+)
+
+type mysqlRepository struct {
+	Conn *sql.DB
+}
+
+// Init will create an object that represent the attachment's Repository interface
+func Init(db *sql.DB) domain.AttachmentRepository {
+	return &mysqlRepository{
+		Conn: db,
+	}
+}
+
+func (r mysqlRepository) CreateAttachment(ctx context.Context, a domain.Attachment) error {
+	query := `INSERT  attachments SET title=?, description=?, name=?, size=?, type=?`
```

Comment:
Seems to have a redundant space character.
```suggestion
	query := `INSERT attachments SET title=?, description=?, name=?, size=?, type=?`
```